### PR TITLE
Update copy on settings

### PR DESCRIPTION
**Pull request by bc-jcruiz**
#### What?

While updating the documentation for settings, the team discovered some confusing names that were used for settings. In this PR I provide alternative names that correlate more with what the user is actually changing via the setting.

#### Tickets / Documentation
N/A

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Comment by bc-jcruiz** [comment]
impact-low.
@BC-EChristensen for review
Thanks @kchu93 for helping navigate through forks!!!

File: schema.json [hunk 1]
```diff
@@ -2579,14 +2579,9 @@
...
       },
-      {
-        "type": "color",
-        "label": "Checkbox button background",
-        "id": "optimizedCheckout-formField-inputControlColor"
-      },
       {
         "type": "heading",
-        "content": "Form checklist"
+        "content": "Payment Methods"
```
Comment (BC-EChristensen) [inline schema.json]: Should be a lowercase 'methods'

**Comment by junedkazi** [comment]
@bc-jcruiz @BC-EChristensen if the changes are ready can we squash the commits and get the PR ready for one final review.

**Comment by mattolson** [comment]
This branch has a lot of unnecessary commits, you should rebase so it contains only your changes.

File: schema.json [hunk 2]
```diff
@@ -2579,14 +2579,9 @@
...
       },
-      {
-        "type": "color",
-        "label": "Checkbox button background",
-        "id": "optimizedCheckout-formField-inputControlColor"
-      },
       {
         "type": "heading",
-        "content": "Form checklist"
+        "content": "Payment methods"
```
Comment (mattolson) [inline schema.json]: This doesn't seem correct.

File: schema.json [hunk 3]
```diff
@@ -2621,6 +2616,11 @@
         "type": "color",
         "label": "Toaster text color",
         "id": "optimizedCheckout-loadingToaster-textColor"
+      },
+      {
+        "type": "color",
+        "label": "Radio button background",
+        "id": "optimizedCheckout-formField-inputControlColor"
```
Comment (mattolson) [inline schema.json]: This change doesn't seem correct, should be moved back.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 ## Draft
 - Remove unnecessary API call to get cookie notification status [#1380](https://github.com/bigcommerce/cornerstone/pull/1380)
+- Update copy on quantity increase/decrease icon Cornerston settings [#1369](https://github.com/bigcommerce/cornerstone/pull/1369)
```
Comment (mattolson) [inline CHANGELOG.md]: Typo

**Pull request closed without merging**